quay/quay
# [PROJQUAY-160] Remove TOS and Privacy pages

### Description of Changes

Removes deprecated CoreOS `Terms of Service` and `Privacy` pages. These were replaced in #242 with footer links to the official Redhat & Openshift policies.

#### Changes:

* Terms of Service page and route is removed.
* Privacy Policy page and route is removed.

#### Issue:

* https://issues.redhat.com/browse/PROJQUAY-160

**TESTING**

* Ensure `/tos` is no longer accessible
* Ensure `/privacy` is no longer accessible
* Ensure no other references to either page still exist.

**BREAKING CHANGE**

n/a

---

## Reviewer Checklist
